bindings/pyroot_experimental/PyROOT/test/TreeHelper.h

```diff
@@ -67,13 +67,13 @@ void CreateTTree(const char *filename, const char *treename, int nentries, int a
 void CreateTNtuple(const char *filename, const char *tuplename, int nentries, int more,
                    const char* openmode)
 {
-   std::stringstream ss;
-   ss << tuplename << "D";
-   auto tuplenamed = ss.str().c_str();
```

Comment:
This is the operation that is illegal. (i.e. retaining the internal address of a temporary object (the result of str()).

Another way to solve the problem is to use:
```
TNtupleD ntupled(ss.str().c_str(), "Test tupled", "x:y:z");
```

Reply:
Can you also guess why it was failing only in that machine? Me and @etejedor have no idea.

Reply:
Was probably just working by chance in the other machines, since as @pcanal pointed out the problem is that a temporary was being passed.